targets/TARGET_STM/mbed_rtx.h

```diff
@@ -19,7 +19,15 @@
 
 #ifndef INITIAL_SP
 
-#if (defined(TARGET_STM32F051R8) ||\
+#if (defined(TARGET_STM32L475VG))
+/* only GCC_ARM and IAR toolchain have the stack on SRAM2 */
+#if (defined(TOOLCHAIN_GCC_ARM) || defined(TOOLCHAIN_GCC_CR) || defined(__IAR_SYSTEMS_ICC__ ))
```

Comment:
I would expect to have also `&& defined(TWO_RAM_REGIONS)` on this line ?

Reply:
ok